src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/attribute.yml

```diff
@@ -0,0 +1,76 @@
+admin_attribute_groups_attributes:
+  path: /attributes
+  methods: GET
+  defaults:
+    _controller: 'PrestaShopBundle:Admin/Sell/Catalog/Attribute:index'
+    _legacy_controller: AdminAttributesGroups
+    redirectQueryParamsToKeep:
+      - 'attributeGroupId'
+#    _legacy_link: AdminAttributesGroups:viewattribute_group
+#    _legacy_parameters:
+#      id_attribute_group: attributeGroupId
+#  requirements:
+#    attributeGroupId: \d+
+
+admin_attribute_groups_attributes_search:
+  path: /attributes
+  methods: POST
+  defaults:
+    _controller: 'PrestaShopBundle:Admin/Sell/Catalog/Attribute:search'
+    _legacy_controller: AdminAttributesGroups
+#    _legacy_link: AdminAttributesGroups:submitFilterattribute_values
+    redirectQueryParamsToKeep:
+      - 'attributeGroupId'
+
+#@todo: implement
+admin_attribute_groups_attributes_create:
```

Comment:
Hmmm, would it make sense to rename route to `admin_attribute_create`? Naming convention says `admin_{resources}_{action}`, but reading your name it seems that it follows `admin_{resources}_{resources}_{action}` naming.